core/commands/files/files.go

```diff
@@ -0,0 +1,707 @@
+package commands
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	gopath "path"
+	"strings"
+
+	cmds "github.com/ipfs/go-ipfs/commands"
+	core "github.com/ipfs/go-ipfs/core"
+	dag "github.com/ipfs/go-ipfs/merkledag"
+	mfs "github.com/ipfs/go-ipfs/mfs"
+	path "github.com/ipfs/go-ipfs/path"
+	ft "github.com/ipfs/go-ipfs/unixfs"
+	u "github.com/ipfs/go-ipfs/util"
+
+	context "github.com/ipfs/go-ipfs/Godeps/_workspace/src/golang.org/x/net/context"
+)
+
+var log = u.Logger("cmds/files")
+
+var FilesCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Manipulate unixfs files",
```

Comment:
good question. i'm really not sure what the cleanest thing to do is. Posix files + dirs we use today across all OSes are unix derived, hence unixfs.

i think this should not be named `mfs`, but `{unix, unixfs, files}`. (`mfs` will be confusing to people unless we really push it and use that name everywhere. i think the other thing -- with mounts -- is likely better target to keep for the `mfs` name anyway. let's just say `ipfs unix` or `ipfs files`)